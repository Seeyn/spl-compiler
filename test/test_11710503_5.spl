struct foo{
    int a;
    float c;
};

struct foo{
    int d;
    float e;
};

int main(){
    int b[10];
    b[1.0] = 10;
}