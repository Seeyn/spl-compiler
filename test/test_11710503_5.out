Error type 15 at Line 6: redefine struct: foo
Error type 12 at Line 13: indexing by non-integer
